src/main/java/uk/gov/hmcts/reform/roleassignment/domain/model/ExistingRoleAssignment.java

```diff
@@ -0,0 +1,11 @@
+package uk.gov.hmcts.reform.roleassignment.domain.model;
+
+import lombok.NoArgsConstructor;
+import lombok.experimental.SuperBuilder;
+
+@SuperBuilder
+@NoArgsConstructor
+public class ExistingRoleAssignment extends Assignment {
+
```

Comment:
this probably is not correct... Both existing and new role assignments are only logical objects and can be created using 1 domain class. You dont need to create two separate classes for same

Reply:
It is required from drools perspective as we have separate set of validation rules for newly requested assignments and existing assignments of actor, hence it is important to keep them separate.

Reply:
yeah,it's required for drools validation as per the new requirnment.